src/styles/blocks/header.scss

```diff
@@ -0,0 +1,76 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
```

Comment:
do not use `*` for `margins` or `font-family`

leave only `box-sizing` here

reset margin for some special tags, elements
move `font-family` to `html` selector

Reply:
move  it to a separate file like `reset.scss`